web_src/js/features/repo-issue-list.js

```diff
@@ -206,6 +206,8 @@ function initArchivedLabelFilter() {
     for (const label of archivedLabels) {
       const id = label.getAttribute('data-label-id');
       toggleElem(label, archivedLabelEl.checked || selectedLabels.includes(id));
+      // also toggle the divider
+      toggleElem($(label).next('.exclusive-scope'), archivedLabelEl.checked || selectedLabels.includes(id));
```

Comment:
1. There is not necessarily a divider
2. I think the condition is wrong. It works in the situation you've tested, but I'm fairly certain that in more complex situations it will hide the divider even though it shouldn't.